docs/source/how_to_guides/parallelization/hpc_scheduler.md

```diff
@@ -0,0 +1,169 @@
+# Parallelizing pipeline runs on HPC systems
+
+This guide shows how to parallelize pipeline runs on {term}`HPC` systems that use job schedulers supported by the Nipoppy tools.
+
+Currently, we have built-in support for the [Slurm](https://slurm.schedmd.com/overview.html) and [SGE](https://en.wikipedia.org/wiki/Oracle_Grid_Engine) job schedulers only.
+However, it may also be possible to use other job schedulers [with some manual work](#support-for-other-job-schedulers).
+
+```{important}
+Although the default template job script is designed to work with minimal user configuration, each {term}`HPC` system is different, and some may require different/additional parameters to be set. See the [Further customization](#further-customization) section for how deeper configuration can be achieved.
+
+If the default Slurm/SGE configurations do not work for you, please consider [opening an issue on our GitHub repository](https://github.com/nipoppy/nipoppy/issues/new/choose) so that we can improve our HPC support.
+```
+
+## Configuring main HPC options
+
+### Global settings
+
+The default global configuration file has two {term}`HPC`-related fields that should be updated as needed:
+
+```{literalinclude} ../../../../nipoppy/data/examples/sample_global_config.json
+---
+linenos: True
+language: json
+emphasize-lines: 4,16-18
+---
+```
+
+#### `HPC_PREAMBLE`
+
+`HPC_PREAMBLE` is a list of Bash commands that should executed at the beginning of **every** job.
+Importantly, there should be a command for activating the Nipoppy Python environment.
+* For {term}`venv` environments, the command would be something like this: `"source <PATH_TO_NIPOPPY_VENV>/bin/activate"`
+* For {term}`conda` environments, the command would instead be something like this: `"source ~/.bashrc; conda activate <NIPOPPY_ENV_NAME>"`
+
+#### `[[HPC_ACCOUNT_NAME]]`
+
+The value for the `[[HPC_ACCOUNT_NAME]]` field in the `SUBSTITUTIONS` dictionary should be set to the account name/ID the job will be associated with.
+By default this will be passed as `--account-name` in Slurm systems and `-q` in SGE systems during job submission.
+This can be left blank if these options are not needed.
+
+```{attention}
+If your HPC system needs flags other than `--account-name` or `-q` need to be set, you will have to modify the template job submission script: see the [Further customization](#further-customization) section for more information.
+```
+
+### Pipeline-specific settings
+
+Job time limit and CPU and memory requests can be configured separately for each pipeline via the HPC config file.
+Look for this file inside the pipeline config directory at {{dpath_pipelines}}`/{bidsification,processing,extraction}/<PIPELINE_NAME>/<PIPELINE_VERSION>` -- it is most likely called `hpc.json` or `hpc_config.json` and should look something like this:
```

Comment:
We can specify the field to look for in the pipeline `config.json` file, in case it is name differently.